designsafe/static/scripts/projects/components/file-categories/file-categories.component.js

```diff
@@ -10,6 +10,10 @@ import { values } from '@uirouter/core';
 
 const getFileUuid = (file) => {
     let promise;
+    if (file.apiParams.fileMgr === 'published') {
```

Comment:
Could this be handled in the state provider?
https://github.com/DesignSafe-CI/portal/blob/87bfcb787b82de2697228b853bebb4cf0fc2276e/designsafe/static/scripts/data-depot/index.js#L951